### ISSUE TITLE
License, code of conduct and other repo admin

### ISSUE THREAD

**Issue by caro401** [codexfelis/oratio#1]
I initially pushed this in a bit of a hurry, and forgot to update the readme!

@danwaterfield I guess we can license this content as some kind of creative commons, maybe [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/deed.en)?

- [x] add license
- [x] add repo description and tags
- [x] configure funding.yml
- [x] add code of conduct
- [x] update the readme
- [x] deploy on a real domain name

**Comment by caro401** [comment]
I think we need a separate license for the code and content - MIT for code and CC BY-SA 4.0 for content I think makes most sense. See https://github.com/github/choosealicense.com#license for an example of doing this. @danwaterfield are you happy with those licenses?

**Comment by caro401** [comment]
Added funding.yml in https://github.com/codexfelis/oratio/commit/8f78a2f8b603eef1f22292d557ec1ec53b048423

**Comment by caro401** [comment]
Added license and things in https://github.com/codexfelis/oratio/commit/b94bf583678daffa1f15290b68896bf5846a14ea

**Comment by caro401** [comment]
Blocked by at least #4 being completed, so we don't go live publicly with a half-written article!

**Comment by caro401** [comment]
is `oratio.codexfelis.dev` the right domain name to use (at least for now?) @danwaterfield? Not sure it's worth buying a separate one?

**Comment by caro401** [comment]
I just remembered [Sembr](https://sembr.org/) exists. @danwaterfield do you 
- like this? (or could at least tolerate using it?
- think it's worth mentioning this in the contributing guidelines, so we encourage people to use it

**Comment by caro401** [comment]
Done the domain name setup - https://oratio.codexfelis.dev, will open a separate issue for considering sembr
